caravel/views.py

```diff
@@ -32,7 +32,8 @@
 from wtforms.validators import ValidationError
 
 import caravel
-from caravel import appbuilder, db, models, viz, utils, app, sm, ascii_art
+from caravel\
```

Comment:
I favor parenthesis over `\` as in

from foo import (
    bar1,
    bar2,
    ...
)